packages/picasso/src/MenuItem/MenuItem.tsx

```diff
@@ -129,52 +133,51 @@ export const MenuItem: OverridableComponent<Props> = forwardRef<
         value={value}
         selected={selected || isOpened}
       >
-        <Container
-          ref={anchorRef}
-          flex
-          direction='column'
-          className={classes.content}
-        >
-          <Container flex alignItems='center'>
-            {isIconWrapperVisible && (
+        <Container ref={anchorRef} flex direction='row'>
+          {avatar && <Container className={classes.avatar}>{avatar}</Container>}
```

Comment:
I would prefer using Container prop
```suggestion
          {avatar && <Container right='xsmall'>{avatar}</Container>}
```

Reply:
Nice catch, I forgot about `right`